guides/common/modules/proc_synchronizing-templates-with-an-existing-repository.adoc

```diff
@@ -4,46 +4,50 @@
 Use this procedure to synchronize templates between your {ProjectServer} and an existing repository.
 
 .Procedure
-. If you want to use HTTPS to connect to the repository and you use a self-signed certificate authority (CA) on your Git server:
-.. Create a new directory under the `/usr/share/foreman/` directory to store the Git configuration for the certificate:
+. Configure connection to the repository:
+.. To use an HTTP proxy to connect to the repository, ensure the proxy is configured on your {ProjectServer}.
+In the {ProjectwebUI}, navigate to *Infrastructure > HTTP Proxies*.
```

Comment:
```suggestion
```
I wouldn't mention this here, choosing proxy fundamentally doesn't differ from choosing repo or branch.
On the other hand, it would probably be worth mentioning that this entire step can be skipped if the user wants to connect to the repository using an HTTP proxy, an HTTPS proxy, or a regular HTTP connection.

Reply:
Ok, I took another look at the whole procedure and tried to streamline it a bit.